src/styles/sidebar/components/annotation-share-control.scss

```diff
@@ -0,0 +1,51 @@
+@use '../../mixins/buttons';
+@use '../../mixins/links';
+@use '../../mixins/panel';
+@use "../../variables" as var;
+
+.annotation-share-control {
+  position: relative;
+}
+
+.annotation-share-panel {
+  @include panel.panel--compact;
+  position: absolute;
+  right: -5px;
+  bottom: 30px;
+  width: 275px;
+  box-shadow: 0px 0px 5px 0px rgba(0, 0, 0, 0.15);
```

Comment:
get the feeling we could have a standard popup shadow... something to think about for pattern lib

Reply:
Absolutely.

Reply:
See: https://github.com/hypothesis/client/issues/1537